sycl/include/sycl/ext/oneapi/properties/properties.hpp

```diff
@@ -187,6 +187,8 @@ properties(PropertyValueTs... props)
     -> properties<typename detail::Sorted<PropertyValueTs...>::type>;
 #endif
 
+using empty_properties_t = decltype(properties{});
```

Comment:
I'd prefer this to go in a separate PR updating existing places where it can be used.

Reply:
Oh, yes, I forgot to delete from here. I'll push a separate PR soon.

Reply:
https://github.com/intel/llvm/pull/11476